libraries/botbuilder-dialogs-adaptive-testing/src/testActions/customEvent.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @module botbuilder-dialogs-adaptive-testing
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TurnContext, TestAdapter, ActivityTypes } from 'botbuilder-core';
+import { TestAction } from '../testAction';
+
+export interface CustomEventConfiguration {
+    name: string;
+    value?: unknown;
+}
+
+export class CustomEvent extends TestAction implements CustomEventConfiguration {
```

Comment:
If you change this class declaration to:

```typescript
export class CustomEvent<T = unknown> extends TestAction implements CustomEventConfiguration {
  ...
  value: T;
}
```

That opens up the ability for users to strongly type instances of the `CustomEvent` class but supports a useful default of `unknown`.